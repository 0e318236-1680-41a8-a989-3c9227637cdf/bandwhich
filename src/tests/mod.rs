pub mod fakes;
pub mod cases;
