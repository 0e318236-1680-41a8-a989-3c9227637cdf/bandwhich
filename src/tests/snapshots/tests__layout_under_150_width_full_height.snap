---
source: src/tests/mod.rs
expression: "&terminal_draw_events_mirror[0]"
---
 Total Rate Up/Down: 0Bps/0Bps                                                                                                                       
┌Utilization by connection──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│Connection                                          Process               Rate Up/Down                                                             │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
│                                                                                                                                                   │
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
┌Utilization by process name─────────────────────────────────────────────┐┌Utilization by remote ip─────────────────────────────────────────────────┐
│Process                                   Rate Up/Down                  ││Remote Address                            Rate Up/Down                   │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
│                                                                        ││                                                                         │
└────────────────────────────────────────────────────────────────────────┘└─────────────────────────────────────────────────────────────────────────┘

